package.json

```diff
@@ -100,7 +99,7 @@
     "pretest": "check-node-version --package",
     "dev": "node webpack_config/server.js",
     "predev": "check-node-version --package",
-    "flow": "flow",
+    "flow": "flow || true",
```

Comment:
No comment-line, so posting here.

We need to keep the exit code as we want CI to fail when flow checks don't pass.